templates/components/routers/python/sandbox.py

```diff
@@ -60,6 +60,14 @@ class FileUpload(BaseModel):
     name: str
 
 
+SUPPORTED_TEMPLATES = [
```

Comment:
Is it just a raw string value? Is there no available constant from the e2b library?